### ISSUE TITLE
Color mode

### ISSUE THREAD

**Issue by svenhb** [LingDong-/linedraw#4]
Your tool is so cool!!!
I made a color version of the fork https://github.com/mitxela/plotterfun but didn't test the result with my plotter up to now - https://grbl-plotter.de/plotterfun-color
Also drag & drop of an images on to the canvas is possible
To do: I can't change the order of the colors (the lightest color first would be nice), output scaled to given mm (somehow a factor of 3.78 is applied...)
![image](https://user-images.githubusercontent.com/19973251/99852610-9fff0d00-2b81-11eb-9018-9aec5c56cb23.png)

**Comment by LingDong-** [comment]
Hi @svenhb, glad that you're enjoying linedraw! 
The particular problem you're asking is probably better directed at the plotterfun author though; lindraw only contains the tracing and hatching algorithms, and I believe the colors and the user interface are features handled by plotterfun.

**Comment by svenhb** [comment]
Yes you are right. I was so exciting and happy :-) and wanted to share my joy with you

**Comment by LingDong-** [comment]
:D Thanks! 
I just tried out plotterfun color. Very cool project!
